commands/server.js

```diff
@@ -16,6 +16,15 @@
  * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
  */
 
-export default {
-    name: 'some item'
-};
+Cypress.Commands.add('setupServer', () => {
+    cy.server({
+        whitelist: (xhr) => {
+            // this function receives the xhr object in question and
+            // will whitelist if it's a GET that appears to be a static resource
+            // return xhr.method === 'GET' && /\.(jsx?|html|css)(\?.*)?$/.test(xhr.url);
```

Comment:
do we need this comment?

Reply:
Some of it is useful, I think. I reduced it.